package/run.sh

```diff
@@ -21,11 +36,19 @@ fi
 
 env "INSTALL_RKE2_ARTIFACT_PATH=${CATTLE_AGENT_EXECUTION_PWD}" installer.sh
 
+if check_target_mountpoint || check_target_ro; then
+    echo "${SA_INSTALL_PREFIX} is ro or a mount point"
+    SA_INSTALL_PREFIX="/opt/rke2-sa"
```

Comment:
Is there a reason why we want to create our own folder i.e. `/opt/rke2-sa` and not just re-use the `/opt/rke2` folder?